system_reports/MACOS/INTEL/osx-xcode-14.0.x.log

```diff
@@ -20,15 +20,12 @@
 * Java:   system
   1.8
   1.8.0.292
-  11
```

Comment:
jenv version did not change, maybe sed is different? it seems that the known issue https://github.com/jenv/jenv/issues/366 came back, right?